docs/api/pytorch.rst

```diff
@@ -9,6 +9,9 @@ pyTorch
 .. autoapiclass:: transformer_engine.pytorch.Linear(in_features, out_features, bias=True, **kwargs)
   :members: forward, set_tensor_parallel_group
 
+.. autoapiclass:: transformer_engine.pytorch.GroupedLinear(in_features, out_features, bias=True, **kwargs)
+  :members: forward, set_tensor_parallel_group
```

Comment:
I don't see `set_tensor_parallel_group` in the generated docs (see [job artifacts](https://github.com/NVIDIA/TransformerEngine/actions/runs/11052631452?pr=1206)). It's strange, since it shows up for `LayerNormMLP`, `MultiHeadAttention`, `TransformerLayer`, but not for `Linear`, `GroupedLinear`, `LayerNormLinear`. I don't see any differences between these cases.

Reply:
I see that this is the case for the generated docs for the other PRs also.